prow/spyglass/viewers/buildlog/buildlog_viewer.go

```diff
@@ -30,13 +31,18 @@ import (
 )
 
 const (
-	name          = "build-log-viewer"
-	title         = "Build Log"
-	priority      = 10
-	byteChunkSize = 1e3 // 1KB
-	lineChunkSize = 100
+	name            = "build-log-viewer"
+	title           = "Build Log"
+	priority        = 10
+	byteChunkSize   = 1e3 // 1KB
+	lineChunkSize   = 100
+	neighborLines   = 5 // number of "important" lines to be displayed in either direction
+	minLinesSkipped = 5
 )
 
+// errRE matches keywords and glog error messages
+var errRE = regexp.MustCompile(`(?i)timed out|error|fail|fatal|panic|^E\d{4} \d\d:\d\d:\d\d\.\d\d\d]`)
```

Comment:
(as discussed, this regex probably should be configurable in https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L223-L237)

Reply:
Since there's no viewer-specific config yet, this will be implemented in a future PR. Added to #9374